packages/json-schema/test/integration/build-schema.test.ts

```diff
@@ -0,0 +1,216 @@
+// Copyright IBM Corp. 2013,2018. All Rights Reserved.
+// Node module: @loopback/json-schema
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {model, property, ModelMetadataHelper} from '@loopback/repository';
+import {modelToJsonDef, toJsonProperty} from '../../src/build-schema';
+import {expect} from '@loopback/testlab';
+
+describe('build-schema', () => {
+  context('exception', () => {
+    it('errors out when "@property.array" is not used on an array', () => {
+      @model()
+      class BadArray {
+        @property() badArr: string[];
+      }
+      expect(() => {
+        modelToJsonDef(BadArray);
+      }).to.throw(/type is defined as an array/);
+    });
+  });
+  describe('modelToJSON', () => {
```

Comment:
Please add an empty lines between `describe` blocks and also between `it` blocks. They server as a visual delimiter and make it easier to spot where on test (case/suite) ends and the next one starts.